modules/core/owner/farm_owner.module

```diff
@@ -17,8 +17,8 @@ function farm_owner_entity_base_field_info(EntityTypeInterface $entity_type) {
   if (in_array($entity_type->id(), ['asset', 'log'])) {
     $field_info = [
       'type' => 'entity_reference',
-      'label' => t('Owner'),
-      'description' => t('Optionally specify who is in charge of this record.'),
+      'label' => t('Owners'),
+      'description' => t('Assign ownership to the selected user(s).'),
```

Comment:
The "(s)" just bugs me, what about "Assign ownership to one or more users"? That's more explicit that one or more can be selected?

Reply:
Good suggestion @paul121 - it does mean a change to BOTH strings, but I guess we're already deciding to do that with parents. Better now than post-stable I guess. I'll change this and re-push shortly...